src/pages/Governance/components/snackbar/FailureSnackbar.tsx

```diff
@@ -26,16 +29,19 @@ export default function FailureSnackbar({
         action={<CloseAction onCloseSnackbar={onCloseSnackbar} />}
       >
         <Typography variant="inherit">
-          Failed with transaction {""}
+          Transaction {""}
           <Link
             component={RRD.Link}
             to={`/txn/${transactionHash}`}
             color="inherit"
             target="_blank"
           >
             {transactionHash}
-          </Link>
-          {"."} Please try again.
+          </Link>{" "}
+          Failed{" "}
+          {data && "vm_status" in data && data.vm_status
+            ? `with ${data.vm_status}`
```

Comment:
Wondering if it's better to add "..." outside the vm_status to make it clear that this part is the message.

Reply:
not completely sure what you mean here..?

Reply:
Sorry, I meant something like this 
```suggestion
            ? `with "${data.vm_status}"`
```